uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/CopyFileRule.java

```diff
@@ -70,6 +70,7 @@ public class CopyFileRule implements TestRule {
             BrokerLTW.BROKER_LTW_APK,
             BrokerLTW.OLD_BROKER_LTW_APK,
             OneAuthTestApp.ONEAUTH_TESTAPP_APK,
+            OneAuthTestApp.OLD_ONEAUTH_TESTAPP_APK,
```

Comment:
I think you can comment out this part and merge this PR. This would make you work on your next task - To add a test case for update scenarios without being dependent on Dean's changes